applications/RomApplication/tests/structural_dynamic_test_files/test_ROM.py

```diff
@@ -12,11 +12,11 @@
 import KratosMultiphysics.kratos_utilities as kratos_utilities
 
 
-
+@KratosUnittest.skipIfApplicationsNotAvailable("StructuralMechanicsApplication")
```

Comment:
maybe you also need the `LinearSolversApp`?

Reply:
I thought you might be using it when solving the structural problem

Reply:
Gotcha. In realiy, the test takes the default linear solver, if the LinearSolversApp is available, it will be taken it from there, but not necessarily...